### PR TITLE
Cleanup legacy dashboards

### LINKED ISSUE #116: [BREAKING CHANGE] Removal of dashboards for legacy engines
OpenEBS is deprecating a number of legacy projects. These legacy projects along with their repositories will be moved to a new CNCF-owned GitHub Org on 8 April 2024 (or soon after).
 
This should not come as a surprise; we began informing the community from Jan 2024 of our work with CNCF TOC to optimize and modernize the project structure. The legacy projects include older data engines, subprojects, unused repositories and build dependencies; and over the last 12 months we have only made minor changes to these projects. The deprecated repositories will move to a new GitHub organization named “openebs-archive”. As the projects are moved, some will be tagged as archived, some will not be marked as archived, but over time they all will be.
 
Deprecated Engines:
1. cStor
2. Jiva
3. NFS Provisioner
4. Device LocalPV
 
Since `openebs/monitoring` repo holds the code related to above engines, as a part of upcoming major release the monitoring stack for these engines would also be deprecated. In the same attempt we have also tried to bump up the respective dependencies, which restricts us from a having a direct upgrade path. It is expected that the upgrades will break.
 
This issue serves as a notice to inform about the breaking change in the coming major release. Users are requested to plan their upgrade paths accordingly.

### PR DESCRIPTION
**Pull request by w3aman**
Fixes #116

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ A set of Grafana dashboards and Prometheus alerts for OpenEBS that can be instal
 
 ## Status
 
-**Beta**. This repository currently supports dashboards and alerts for `cStor`, `Jiva`, `LVM LocalPV` OpenEBS storage engines.
+**Beta**. This repository currently supports dashboards and alerts for `mayastor`, `LocalPV LVM`, `LocalPV ZFS` OpenEBS storage engines.
```
Comment (avishnu) [inline README.md]: Mayastor

**Review by avishnu — APPROVED** [review]

**Comment by avishnu** [comment]
Please fix jsonnet lint error.

**Comment by niladrih** [comment]
It might be useful if this repo tracks the version from the umbrella chart.
So, let's set both appVersion and version to 4.0.0? WDYT @avishnu @tiagolobocastro @Abhinandan-Purkait?

**Comment by avishnu** [comment]
> It might be useful if this repo tracks the version from the umbrella chart. So, let's set both appVersion and version to 4.0.0? WDYT @avishnu @tiagolobocastro @Abhinandan-Purkait?

I think, monitoring should follow its own version. Not every release need to have monitoring changes. Wdyt?

**Comment by niladrih** [comment]
Alright, sounds good! That brings monitoring to appVersion 3.0.0 and version 1.0.0. A major bump for the semvers.

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ A set of Grafana dashboards and Prometheus alerts for OpenEBS that can be instal
 
 ## Status
 
-**Beta**. This repository currently supports dashboards and alerts for `cStor`, `Jiva`, `LVM LocalPV` OpenEBS storage engines.
+**Beta**. This repository currently supports dashboards and alerts for `Mayastor`, `LocalPV LVM`, `LocalPV ZFS` OpenEBS storage engines.
```
Comment (niladrih) [inline README.md:9]: Should this be 'Replicated PV Mayastor' instead?
cc: @avishnu

**Review by niladrih — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (orville-wright) [reply README.md:9]: Yes, @niladrih you are correct. Mayastor = "Replicated PV Mayastor"

File: contribute/release.md [hunk 3]
```diff
@@ -7,17 +7,17 @@ To generate dashboards and alert rules for monitoring helm chart and release a n
 
 2. Update the dashboards and rules folder in [build.sh](../jsonnet/openebs-mixin/build.sh). The auto generated json files will be inside this folder.  
    
-   For example: If you add dashboards and alert rules for mayastor in openebs-mixin, then add mayastor as a folder name in build.sh file. 
+   For example: If you add dashboards and alert rules for Mayastor in openebs-mixin, then add mayastor as a folder name in build.sh file. 
    ```
 	generateDashboards(){
 		. . . 
-		dashboardsFolder=(cStor jiva localPV npd lvmLocalPV ndm mayastor)
+		dashboardsFolder=(npd lvmLocalPV zfsLocalPV mayastor)
```
Comment (Abhinandan-Purkait) [inline contribute/release.md:14]: What is npd?

**Review by Abhinandan-Purkait — APPROVED** [review]
LGTM.

@avishnu @tiagolobocastro @niladrih  Now that we are doing a major bump should we change the chart name from `openebs-monitoring` to `monitoring` ?

File: contribute/release.md (hunk 3 above)
Comment (w3aman) [reply contribute/release.md:14]: npd -> node-problem-detector

with @avishnu we have this thought to change `openebs-monitoring` to `monitoring`. Should i update it ?

Comment (avishnu) [reply contribute/release.md:14]: Ya, I think we should change it to monitoring? Can you list down where all the changes would reflect?

Comment (w3aman) [reply contribute/release.md:14]: 1. Chart name and we are using this name for prepending the resource names like following.

2. pod names after installing monitoring helm chart

```
aman@aman-mbp:charts$ ko get po | grep openebs-monitoring
alertmanager-openebs-monitoring-kube-pr-alertmanager-0   2/2     Running   0          4h34m
openebs-monitoring-grafana-fb9cc8cd4-l9sdx               3/3     Running   0          4h34m
openebs-monitoring-kube-pr-operator-bd7fdd6cd-vq5x8      1/1     Running   0          4h34m
openebs-monitoring-kube-state-metrics-c478cc764-9d7hm    1/1     Running   0          4h34m
openebs-monitoring-prometheus-node-exporter-64zvh        1/1     Running   0          4h34m
openebs-monitoring-prometheus-node-exporter-74wxb        1/1     Running   0          4h34m
openebs-monitoring-prometheus-node-exporter-g454f        1/1     Running   0          4h34m
openebs-monitoring-prometheus-node-exporter-ghndk        1/1     Running   0          4h34m
prometheus-openebs-monitoring-kube-pr-prometheus-0       2/2     Running   0          4h34m
```

3. service-monitor names

```
aman@aman-mbp:charts$ k get servicemonitor -n openebs
NAME                                                 AGE
openebs-monitoring-grafana                           4h36m
openebs-monitoring-kube-pr-alertmanager              4h36m
openebs-monitoring-kube-pr-apiserver                 4h36m
openebs-monitoring-kube-pr-coredns                   4h36m
openebs-monitoring-kube-pr-kube-controller-manager   4h36m
openebs-monitoring-kube-pr-kube-etcd                 4h36m
openebs-monitoring-kube-pr-kube-proxy                4h36m
openebs-monitoring-kube-pr-kube-scheduler            4h36m
openebs-monitoring-kube-pr-kubelet                   4h36m
openebs-monitoring-kube-pr-operator                  4h36m
openebs-monitoring-kube-pr-prometheus                4h36m
openebs-monitoring-kube-state-metrics                4h36m
openebs-monitoring-lvmlocalpv                        4h36m
openebs-monitoring-mayastor                          4h36m
openebs-monitoring-prometheus-node-exporter          4h36m
```

4. services
aman@aman-mbp:charts$ k get svc -n openebs | grep openebs-monitoring
```
openebs-monitoring-grafana                    NodePort    10.109.38.93     <none>        80:32515/TCP                                            4h37m
openebs-monitoring-kube-pr-alertmanager       NodePort    10.102.59.49     <none>        9093:30903/TCP,8080:31936/TCP                           4h37m
openebs-monitoring-kube-pr-operator           ClusterIP   10.105.103.10    <none>        443/TCP                                                 4h37m
openebs-monitoring-kube-pr-prometheus         NodePort    10.99.162.154    <none>        9090:32514/TCP,8080:31856/TCP                           4h37m
openebs-monitoring-kube-state-metrics         ClusterIP   10.105.215.88    <none>        8080/TCP                                                4h37m
openebs-monitoring-prometheus-node-exporter   ClusterIP   10.109.138.28    <none>        9100/TCP
```

**Review by niladrih — APPROVED** [review]

**Pull request merged**